### PR TITLE
Reworded a note about custom constraints

### PR DESCRIPTION
**Pull request by javiereguiluz**
This replaces #6610.

### CODE REVIEW

**Comment by Jean85** [comment]
LGTM :+1: 

Status: reviewed

File: validation/custom_constraint.rst [hunk 1]
```diff
@@ -202,9 +202,9 @@ load this service from the container.
 
 .. note::
 
-    In earlier versions of Symfony, the tag required an ``alias`` key (usually set
-    to the class name). This is still allowed your constraint's ``validateBy()``
-    method can return this alias (instead of a class name).
+    In earlier versions of Symfony, the tag required an ``alias`` key (usually
+    set to the class name). This ``alias`` is now optional, but if you define
+    it, your constraint's ``validateBy()`` method must return the same value.
```
Comment (xabbuh) [inline validation/custom_constraint.rst]: Looks like we always had a typo here. The method's name is `validatedBy()`.

**Comment by xabbuh** [comment]
@javiereguiluz This is targeted for 2.7, right?

**Comment by javiereguiluz** [comment]
Yes! I've changed the base to 2.7. Thanks.

**Review by HeahDude — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**